spring-cloud-generator/generate-all.sh

```diff
@@ -33,7 +37,9 @@ while IFS=, read -r library_name googleapis_location coordinates_version googlea
     -g $group_id \
     -p $PROJECT_VERSION \
     -f $googleapis_location \
-    -x $googleapis_commitish 2>&1 | tee tmp-generate-one-output || save_error_info $library_name
+    -F $monorepo_folder \
```

Comment:
Nit: should we pick a different lowercase letter for this flag, to avoid confusion with the lowercase f flag?

Reply:
I see the point - I'll just use another flag letter.